### PR TITLE
u3d/install: convert Windows paths to ruby paths when treating U3D_EXTRA_PATHS

### LINKED ISSUE #383: U3D_EXTRA_PATHS is improperly interpreted on Windows
Commands:

> u3d install 2019.2.12f1 --installation_path "C:\Program Files\Unity\Unity_2019.2.12f1"

> u3d install 2019.2.12f1 -p Universal-Windows-Platform --installation_path "C:\Program Files\Unity\Unity_2019.2.12f1"

Log:

```
E:\workspace\test>u3d install 2019.2.12f1 --installation_path "C:\Program Files\Unity\Unity_2019.2.12f1" 
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/u3d-1.2.1/lib/u3d/installation.rb:324: warning: Win32API is deprecated after Ruby 1.9.1; use fiddle directly instead
Root privileges are required
Unity 2019.2.12f1 is already downloaded
--------------------------------------------
--- Installing Unity 2019.2.12f1 (Unity) ---
--------------------------------------------
Installing with //Apps/Unity/Windows/2019.2.12f1/UnitySetup64.exe
$ //Apps/Unity/Windows/2019.2.12f1/UnitySetup64.exe -UI=reduced /S /D=C:\Program Files\Unity\Unity_2019.2.12f1
No pty implementation found. Falling back to popen. Output might be buffered
Successfully installed Unity 2019.2.12f1
[Pipeline] isUnix
[Pipeline] bat

E:\workspace\test>u3d install 2019.2.12f1 -p Universal-Windows-Platform --installation_path "C:\Program Files\Unity\Unity_2019.2.12f1" 
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/u3d-1.2.1/lib/u3d/installation.rb:324: warning: Win32API is deprecated after Ruby 1.9.1; use fiddle directly instead
Please install Unity before any of its packages
```

### PR DESCRIPTION
**Pull request by lacostej**
### Pull Request Checklist

- [x] My pull request has been rebased on master
- [x] I ran `bundle exec rspec` to make sure that my PR didn't break any test
- [x] I ran `bundle exec rubocop` to make sure that my PR is inline with our code style
- [x] I have read the [code of conduct](https://github.com/DragonBox/u3d/blob/master/CODE_OF_CONDUCT.md)

### Pull Request Description

The documentation states that U3D_EXTRA_PATHS are windows style.

Fixes #383

### CODE REVIEW

**Comment by lacostej** [comment]
I didn't this :)

**Comment by lacostej** [comment]
👍   I let you test&fix this then.

**Comment by lacostej** [comment]
@niezbop 

image

image

**Comment by niezbop** [comment]
@lacostej Cool! I was pretty sure `File.expand_path` would work!

**Comment by niezbop** [comment]
> Not the platform you are looking for

💯

File: spec/u3d/installer_spec.rb [hunk 1]
```diff
@@ -88,6 +88,25 @@ class DummyInstaller < U3d::BaseInstaller
           expect(installer.installed_sorted_by_versions).to eq(sorted_installed)
         end
       end
+
+      describe ".extra_installation_paths" do
+        it "converts paths to ruby paths" do
```
Comment (niezbop) [inline spec/u3d/installer_spec.rb]: I think it'd be better to have an intermediary `describe 'On Windows', if: WINDOWS do ... end` and `describe 'On other platforms', unless: WINDOWS do ... end` than handling the logic ourselves in the test. That's the convention we follow already for platform-dependent testing (see following test case).

**Pull request merged**